src/main/java/com/digitalasset/testing/cucumber/utils/TableUtils.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2019 Digital Asset (Switzerland) GmbH and/or its affiliates
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its affiliates.
+// All rights reserved.
+
+package com.digitalasset.testing.cucumber.utils;
+
+import com.daml.ledger.javaapi.data.Record;
+import com.digitalasset.daml_lf.DamlLf1;
+
+import java.util.*;
+
+import static com.digitalasset.testing.Dsl.*;
+
+public class TableUtils {
+  public static Record fieldsToArgs(List<String> args, List<DamlLf1.FieldWithType> fields) {
+    if (args.size() != fields.size()) {
+      throw new IllegalArgumentException(
+          "Wrong number of actual arguments: " + args.size() + "(formal: " + fields.size() + ")");
```

Comment:
minor: you might want to put a space after `args.size()` :)

Reply:
Yes.